cmd/otel-allocator/README.md

```diff
@@ -4,6 +4,135 @@ The TargetAllocator is an optional separately deployed component of an OpenTelem
 distribute targets of the PrometheusReceiver on all deployed Collector instances. The release version matches the
 operator's most recent release as well.
 
+In essence, Prometheus Receiver configs are "stolen" and replaced with a http_sd_configs directive that points to the 
```

Comment:
There are two things that happen here: one is the overriding of the service discovery in the collector's scrape config, and the other thing is the target allocator can add new scrape configs. I see them as two distinct actions because one can use the target allocator without the prometheus CR functionality.

Reply:
I'll have a look at rewriting this section this weekend to clarify.

Reply:
Updated